src/js/game/hud/parts/sandbox_controller.js

```diff
@@ -105,7 +105,7 @@ export class HUDSandboxController extends BaseHUDPart {
         this.root.hubGoals.upgradeImprovements[id] = improvement;
         this.root.signals.upgradePurchased.dispatch(id);
         this.root.hud.signals.notification.dispatch(
-            "Upgrade '" + id + "' is now at level " + this.root.hubGoals.upgradeLevels[id],
+            "Upgrade '" + id + "' is now at level " + (this.root.hubGoals.upgradeLevels[id] + 1),
```

Comment:
Change looks good. It makes sense that the change is applied here as it's purely a display thing.

I would suggest that you change the word `level` to `tier` to keep it consistent with the rest of the interfaces.

Reply:
I agree. Changed it to say "tier".